turbinia/client.py

```diff
@@ -234,8 +244,15 @@ def get_task_data(self, instance, _, __, days=0, task_id=None,
     return self.redis.get_task_data(instance, days, task_id, request_id)
 
 
-class TurbiniaServer(TurbiniaClient):
+class TurbiniaServer(object):
   """Turbinia Server class."""
+
```

Comment:
Add tast_manager to Attributes docstring

Reply:
Done.